examples/ensemble/plot_ensemble_oob.py

```diff
@@ -0,0 +1,91 @@
+"""
+=========================================================
+OOB Errors for Random Forests and Extra Trees Classifiers
+=========================================================
+
+The ``RandomForestClassifier`` and ``ExtraTreesClasifier`` are trained using
```

Comment:
Actually, maybe @glouppe wants to comment on this. I'm not sure if people use ExtraTrees with bootstrap=True.
Btw, there is a typo in ExtraTreesClassifier here.